Modules/Blueprints/include/selxBlueprint.h

```diff
@@ -64,6 +64,8 @@ class Blueprint : public itk::DataObject
 
   const BlueprintImpl & GetBlueprintImpl( void ) const; 
 
+  void SetBlueprint( BlueprintImplPointer blueprint );
```

Comment:
I don't think we need a SetBlueprint. The current develop hasn't and everything works.